destinations:
  "1":
    drop_pose:  {x: 1.981, y: -0.017, yaw: -2.984}
  "2":
    drop_pose:  {x: 1.928, y:  0.483, yaw: -1.449}
  "3":
    drop_pose:  {x: 2.065, y: -0.425, yaw:  1.640}

home: {x: 0.024, y: -0.009, yaw: -0.005}

unload:
  pwm_open:  1000.0
  pwm_close: 1500.0
  hold_sec:  3.0

topics:
  dest:     "/unload/destination"             # std_msgs/Int32
  status:   "/unload/status"                  # std_msgs/String
  dump_pwm: "/turtlebot3_dumpbox/servo/pwm"   # std_msgs/Float32
  returned: "/unload/returned"                # std_msgs/Empty
